pkg/views/workspace/selection/repository.go

```diff
@@ -44,28 +48,35 @@ func selectRepositoryPrompt(repositories []apiclient.GitRepository, projectOrder
 		os.Exit(1)
 	}
 
+	// Return either the choice or navigation
 	if m, ok := p.(model[string]); ok && m.choice != nil {
```

Comment:
Not sure where to comment this exactly so putting it here:

![Screenshot 2024-10-07 at 12 23 39](https://github.com/user-attachments/assets/f667e0e4-ece5-4550-a3e5-3ebe30739a7e)

We should always add the `Load more` item when the user filters results (if the current list length is equal to Page * PageSize - if it is less, it means there are no more items in the API). This will let the users know that their result might still not be loaded into memory.

Reply:
Not finding a way, seems bubbletea doesnt provides a way to alter the filter results.

Reply:
yup, didnt found a way again.

Reply:
https://github.com/charmbracelet/bubbles/blob/master/list/list.go#L157

Seems there is a way to do it by passing a custom `FilterFunc` to `Filter`.

The custom function can call [list.DefaultFilter](https://github.com/charmbracelet/bubbles/blob/master/list/list.go#L89) and then append `[{index: LAST_INDEX}]` to the list of ranks. This will then display the Load More item in the filtered list.

Reply:
yay, that worked. Thanks for being a kind reviewer..